src/Resources/views/admin/activities/list.html.twig

```diff
@@ -57,7 +57,7 @@
                                         {{ 'cmf_activities_details' | trans({}, 'admin') }}
                                     </a>
                                 </td>
-                                <td width="160">{{ activity.activityDate.formatLocalized('%x %X') }}</td>
+                                <td width="160">{{ activity.activityDate.isoFormat('L LTS') }}</td>
```

Comment:
Should we maybe provide a infos somewhere what the L and LTS stands for? 
https://momentjs.com/docs/